tidb_binlog/binlog_driver/example/mysql/mysql.go

```diff
@@ -22,8 +22,8 @@ import (
 	"github.com/Shopify/sarama"
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/ngaut/log"
-	"github.com/pingcap/tidb-tools/binlog_driver/reader"
-	pb "github.com/pingcap/tidb-tools/binlog_proto/go-binlog"
+	"github.com/pingcap/tidb-tools/tidb_binlog/binlog_driver/reader"
```

Comment:
s/binlog_driver/driver/ ?  
if it's a sub path of tidb_binlog now

Reply:
ok